README.md

```diff
@@ -202,23 +208,39 @@ Returns the transactions collection by the lock or type script.
 #### Parameters
 
     search_key:
-        script - Script
+        script - Script, supports prefix search when group_by_transaction is false
         scrip_type - enum, lock | type
         filter - filter cells by following conditions, all conditions are optional
             script: if search script type is lock, filter cells by type script, and vice versa
             block_range: [u64; 2], filter cells by block number range, [inclusive, exclusive]
+        group_by_transaction - bool, optional default is `false`, if group_by_transaction is set to true, the returning objects will be grouped by the  x hash
     order: enum, asc | desc
     limit: result size limit
     after_cursor: pagination parameter, optional
 
 
 #### Returns
 
-    objects - transactions
+    objects - enum, ungrouped TxWithCell | grouped TxWithCells
```

Comment:
Why not adding a new RPC for the grouped result?

Reply:
Similar to the ckb rpc, which returns data in a different format by parameter, rather than by a different rpc name, for example, in ckb, we are using `get_transaction(tx_hash, verbosity)`, rather than `get_transaction(tx_hash)` and `get_transaction_in_raw(tx_hash)`